docs/1.guide/2.event-handler.md

```diff
@@ -66,38 +64,36 @@ Any of above values could also be wrapped in a [`Promise`](https://developer.moz
 **Example:** Send HTML response:
 
 ```js
-app.use(eventHandler(async (event) => "<h1>Hello world!</h1>"));
+app.use(defineEventHandler(async (event) => "<h1>Hello world!</h1>"));
 ```
 
 **Example:** Send JSON response:
 
 ```js
 app.use(
   "/api",
-  eventHandler(async (event) => ({ url: event.node.req.url })),
+  defineEventHandler(async (event) => ({ url: event.node.req.url })),
 );
 ```
 
 **Example:** Send a promise:
 
 ```js
-app.use(
-  defineEventHandlers(async (event) => {
-    return new Promise((resolve) => {
-      setTimeout(() => {
-        resolve({ url: event.node.req.url });
-      }, 1000);
-    });
-  }),
-);
+app.use(defineEventHandlers(async (event) => {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve({ url: event.node.req.url });
+    }, 1000);
+  });
+}));
```

Comment:
As far as I saw all pages, it seems that line break between `app.use(` and `defineEventHandler` is inserted only when first argument is path string:

```ts
app.use(
  "/",
  defineEventHandlers(async (event) => {
    // ...
  }),
)
```

So, for consistency, I remove line break in some sample code:

```ts
app.use(defineEventHandlers(async (event) => {
  // ...
}))
```